src/styles/classes.scss

```diff
@@ -163,6 +166,26 @@ button {
   }
 }
 
+.hidden {
+  display: none;
+}
+
+.popup-animated {
```

Comment:
This is a sort of cool animation I found from a different repo, lmk what you think.  I think it can be sort of comic-book-like and works really well for small components.  For something full screen like this modal I'm not 1000% sold, but I think it's better than nothing

Reply:
Cool, I'll check it out.